### PR TITLE
fix: changing a link in /developer/keyboards/advanced/step-5 

### PR DESCRIPTION
**Pull request by Meng-Heng**
changing the link title 'Organizing the keyboard files' 
changing the step number according to the link title.

> Following the file layout documented in [Organizing the Keyboard
> Files](step-4), your keyboard layout should be in a folder similar

### CODE REVIEW

File: developer/engine/android/16.0/index.php
Comment (mcdurdin) [inline developer/engine/android/16.0/index.php:1]: This file needs to be restored

File: knowledge-base/kb0033.md
Comment (mcdurdin) [inline knowledge-base/kb0033.md:1]: This file also needs to be restored

**Review by mcdurdin — CHANGES_REQUESTED** [review]
Thanks for the PR @Meng-Heng. Once you restore those two files which are not part of the fix, I can merge this PR

File: knowledge-base/kb0033.md
Comment (Meng-Heng) [reply knowledge-base/kb0033.md:1]: I got the files restored but I got an issue while trying to push to my origin

> fatal: unable to access 'https://github.com/Meng-Heng/help.keyman.com.git/': Could not resolve host: github.com

I will tackle this tomorrow

**Review by mcdurdin — APPROVED** [review]

File: developer/engine/android/16.0/index.php [hunk 1]
```diff
@@ -36,7 +36,7 @@
 <h2 id="See_also" name="See_also">See also</h2>
 
 <ul>
-  <li><a href="/developer/engine/android/15.0/" title="Keyman Engine for Android 15.0">Keyman Engine for Android 15.0</a></li>
+  <li><a href="/developer/engine/android/14.0/" title="Keyman Engine for Android 15.0">Keyman Engine for Android 15.0</a></li>
```
Comment (mcdurdin) [inline developer/engine/android/16.0/index.php]:
```suggestion
  <li><a href="/developer/engine/android/15.0/" title="Keyman Engine for Android 15.0">Keyman Engine for Android 15.0</a></li>
```

**Review by mcdurdin — APPROVED** [review]

**Pull request merged**